azure/group.sh

```diff
@@ -36,12 +36,13 @@ fn azure_group_get_names() {
 # azure_group_exists returns "0" if a resource group
 # already exists, "1" otherwise.
 fn azure_group_exists(name) {
-	group_name <= az group show --name $name | jq -r ".name"
-
+	group_name, err <= az group show --name $name | jq -r ".name"
```

Comment:
This is not an "err" it is the status code of the execution, I would call it "status" so people don't get confused =).

By the way, this stopped working ? I'm pretty sure that when the resgroup does not exist the status code was not different from "0" (just curious, checking this way adds extra safety)

Reply:
renamed to status :)

about double check

```
λ> group_name, status <= az group show --name kmelo
Resource group 'kmelo' could not be found.
λ> echo $status
3
λ>  
```
thankss